scripts/config

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source _common.sh
+source /usr/share/yunohost/helpers
+
+ynh_abort_if_errors
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+encryption_allowed=$(ynh_app_setting_get --app=$app --key=encryption_allowed)
```

Comment:
As I understand what is stated in the example `config_panel.toml`: https://github.com/YunoHost/example_ynh/blob/master/config_panel.toml.example#L162-L165

```toml
        ## Note that this feature only works with relatively simple cases
        ## such as `KEY: VALUE`, but won't properly work with
        ## complex data structures like multilin array/lists or dictionnaries.
        ## It also doesn't work with XML format, custom config function call, php define(), ...
```

You should go with a getter and a setter in the `script/config` panel. If I were you, for the setter, I'd go for a simple:

```bash
set__encryption_allowed() {
    ynh_app_setting_set --app=$app --key=encryption_allowed --value=$encryption_allowed
    ynh_add_config --template="../conf/config.yaml" --destination="$final_path/config.yaml"

    chmod 400 "$final_path/config.yaml"
    chown $app:$app "$final_path/config.yaml"
}
```

Reply:
Thanks, this helped me to understand. I'll fix this and use same structure as in mautrix_whatsapp_ynh